client/platform/web-girder/views/RunTrainingMenu.vue

```diff
@@ -71,6 +78,19 @@ export default defineComponent({
       trainingOutputName.value = null;
     }
 
+    const openTxt = async () => {
+      const data = await openFromDisk(TxtType);
+      if (!data.canceled && data.fileList) {
+        const reader = new FileReader();
+        reader.onload = (evt) => {
+          labelText.value = evt.target?.result as string;
+        };
+        reader.readAsText(data.fileList[0]);
+        // eslint-disable-next-line prefer-destructuring
+        labelFile.value = data.fileList[0];
```

Comment:
```suggestion
        [labelFile.value] = data.fileList;
```

This will resolve the eslint warning so you can remove the disable comment.

Reply:
should be set now